packages/blade/src/components/Amount/amountTokens.ts

```diff
@@ -43,53 +43,275 @@ const amountLineHeights: Record<
   'title-medium': 600,
 } as const;
 
-type CurrencyPrefixMapping = {
-  [key: string]: {
-    'currency-symbol': string;
-    'currency-code': string;
-  };
-};
```

Comment:
Removed this so that I can do `type Currency = keyof typeof currencyPrefixMapping` to get all valid currencies.